ml/Config.cc

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "Config.h"
+#include "ml-private.h"
+
+using namespace ml;
+
+/*
+ * Global configuration instance to be shared between training and
+ * prediction threads.
+ */
+Config ml::Cfg;
+
+/*
+ * Initialize global configuration variable.
+ */
+void Config::readMLConfig(void) {
```

Comment:
TODO:

- add min/max/default values for end-users.
- explicity enable/disable ML configuration option.

Reply:
Will update the rest of the default values in a follow up commit.